remediation-service/handler/handler.go

```diff
@@ -433,6 +448,18 @@ func (r *Remediation) sendActionTriggeredEvent(actionTriggeredEventData keptn.Ac
 
 func (r *Remediation) sendRemediationStatusChangedEvent(action *v0_1_4.RemediationActionsOnOpen, actionIndex int) error {
 
+	triggeredID := ""
+	remediations, err := getRemediationsByContext(r.Keptn.KeptnContext, r.Keptn.Event)
+	if err != nil {
+		r.Keptn.Logger.Error("could not retrieve open remediation: " + err.Error())
+	}
+
+	for _, remediation := range remediations {
+		if remediation.Type == keptn.RemediationTriggeredEventType {
+			triggeredID = remediation.EventID
```

Comment:
should we "break" out of the loop once we have found the ID?

Reply:
good point, I have added the break